MetaMask/metamask-extension
# fix: Fix #25035 - Ensure toasts only display on the home screen

## **Description**

Ensures toasts only display on the home screen so that they do not display over confirmations

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25187?quickstart=1)

## **Related issues**

Fixes: #25035

## **Manual testing steps**

1. Hardcode the privacy policy toast to always display in `ui/pages/routes/routes.component.js`
2. Open MetaMask manually, see toast
3. Go to the test dapp, press "Send" 
4. Don't see policy

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
